nexus/types/src/external_api/params.rs

```diff
@@ -8,11 +8,14 @@
 use crate::external_api::shared;
 use base64::Engine;
 use chrono::{DateTime, Utc};
-use omicron_common::api::external::{
-    AddressLotKind, BfdMode, ByteCount, Hostname, IdentityMetadataCreateParams,
-    IdentityMetadataUpdateParams, InstanceCpuCount, IpNet, Ipv4Net, Ipv6Net,
-    Name, NameOrId, PaginationOrder, RouteDestination, RouteTarget,
-    SemverVersion,
+use omicron_common::api::{
+    external::{
+        AddressLotKind, BfdMode, ByteCount, Hostname,
+        IdentityMetadataCreateParams, IdentityMetadataUpdateParams,
+        InstanceCpuCount, IpNet, Ipv4Net, Ipv6Net, Name, NameOrId,
+        PaginationOrder, RouteDestination, RouteTarget, SemverVersion,
+    },
+    internal::shared::ImportExportPolicy,
```

Comment:
I guess this suggests that `ImportExportPolicy` is part of the external API and so should be in `api::external`?

Reply:
Moved it to `api::external` as suggested.